src/apply-networking-linux.py

```diff
@@ -285,6 +300,7 @@ def apply_network_config(self, network_data, reset_to_dhcp=False):
             addr_add_cmd = base_cmd + ["addr", "add",
                                        ip_address + "/" + prefixlen,
                                        "dev", os_link_name]
+            LOG("Executing: %s" % " ".join(addr_add_cmd))
```

Comment:
you should remove all the executing logs in the execute_process if you want that information.